fastapi_websocket_rpc/websocket_rpc_client.py

```diff
@@ -162,9 +165,6 @@ async def __aenter__(self):
 
     async def __aexit__(self, *args, **kwargs):
         await self.close()
-        # close context of underlying socket
```

Comment:
What about closing the socket on exit?
You close it if an exception is thrown; but what about regular scope termination ?

Reply:
`self.ws.close()` closes it. (`self.ws` wraps the socket and proxies the `close` method)